ios/MaasCore/Sources/MaasCore/Api/Utils/Error+.swift

```diff
@@ -0,0 +1,11 @@
+extension CoreBinary.Error {
```

Comment:
Is a filename like `Error+.swift` typical?

Reply:
Maybe Error+Extensions? 🤔  cc @mikasdo